src/build/arg/mod.rs

```diff
@@ -118,6 +117,11 @@ impl<'help> Arg<'help> {
     pub fn get_possible_values(&self) -> Option<&[&str]> {
         self.possible_vals.as_deref()
     }
+
+    /// Get the index of this argument, if any
+    pub fn get_index(&self) -> Option<u64> {
+        self.index.clone()
```

Comment:
Why is the clone needed here?

Reply:
I'll remove it when I get to clippy check